src/doc/en/developer/review.rst

```diff
@@ -180,14 +180,15 @@ Sage Release Manager uses to make releases. Here it is as of 2023:
 **Beta Release Stage**: For preparing a new beta release or the first release
 candidate, all positively reviewed PRs with the forthcoming release
 milestone are considered. PRs that have dependencies not merged yet are ignored.
-The Release Manager merges PRs in batches of 10 to 20 PRs, taking the
+The Release Manager merges PRs in batches of 10 to 20 PRs, without taking the
 PR priority into account. If a merge conflict of a PR to the Release
 Manager's branch occurs, the PR is set back to "needs work" status by the
```

Comment:
If this is true (@vbraun ?), then the "without ..." clause, which sounds negative, may just be deleted.

Reply:
The release management script does not consider PR priority. Anyway the "without ..." clause seems redundant.

Reply:
Thanks.